DB/projects.json

```
@@ -3555,5 +3555,21 @@
         "tech": ["html", "css"]
       }
     ]
+  },
+  {
+    "github_username": "Gourav2609",
+    "Projects": [
+      {
+        "github_url": "https://github.com/Gourav2609/TO_DO_LIST",
+        "title": "To_Do_List_App",
+        "description": "Learned using React and different React Hooks",
+        "tech": ["React"]
```

convert to lowercase 

```suggestion
        "tech": ["react"]
```